### ISSUE TITLE
"Attach file" is fixed to screen

### ISSUE THREAD

**Issue by rugk** [wekan/wekan#1540]
## Reproduce
1. open "attach file".
2. Scroll.

What happens: The window is fixed when I scroll.
What should happen: it should be attached at it's "parent" "window".

![attachfile](https://user-images.githubusercontent.com/11966684/37524442-242b6f7c-292a-11e8-8bc5-5b7136050019.gif)

**Comment by xet7** [comment]
@rugk 

Fixing this would create usability problem, that I would not see attach file popup is open all the time, if I just want to look around. Attach file popup would just scroll away. It would be problematic on big boards.
